bedrock/thunderbird/templates/thunderbird/includes/top-menu.html

```diff
@@ -0,0 +1,25 @@
+<nav id="nav-main" role="navigation">
+  <span class="toggle" role="button" aria-controls="nav-main-menu" aria-expanded="false" tabindex="0">{{_('Menu')}}</span>
```

Comment:
This is visible in the UI but probably should not be. Also, I do not believe it should have the `role="button` as it does not act like a button and is static text.